src/legacy_wrappers/collection.js

```diff
@@ -238,17 +238,6 @@ module.exports.makeLegacyCollection = function (baseClass) {
       return maybeCallback(super.indexes(options), callback);
     }
 
-    insert(docs, options, callback) {
```

Comment:
There's typescript changes that should be made here as well.  Locally can you point this package's depenency at your local copy of the driver checked out to main and recompiled then double check that the typescript is all happy

Reply:
Good catch, done.  You'll notice that I also fixed `CursorCloseOptions` since that was no longer defined on `main` either.

Reply:
Insert, update, and remove are still there